doc/adr/0011-concept-exception-handling-backend.md

```diff
@@ -0,0 +1,33 @@
+# 10. How to use exceptions in the backend
+
+Date: 2024-06-28
+
+## Status
+
+Accepted
+
+## Context
+So far we used different approaches when to use Optionals, throw checked exceptions or unchecked exceptions. This also led
```

Comment:
I would remove the topic with optionals since the ADR is related to the exception handling. That we throw application exceptions in some cases when we get an optional from the db layer is in my opinion not part of this ADR

Reply:
My perspective is that Optionals are a kind of checked exception in this case and therefore should be part of this discussion.